main.tf

```diff
@@ -3,7 +3,11 @@ provider "aws" {
   version = "~> 2.15"
 }
 
-resource "aws_instance" "ibrah-useast1a" {
+resource "aws_instance" "ibrah-useast-1a" {
```

Comment:
The availability zone here doesn't seem to be persisted to the aws api, which sets it to `useast-1b`. Gonna have to figure out how to do that later.